### PR TITLE
Cleanup readme files

### PR DESCRIPTION
**Pull request by jesusreal**
Confirm these statements before you submit your pull request:

- [x] I have read and submitted the required [Contributor Licence Agreements](https://github.com/kyma-project/community/blob/master/CONTRIBUTING.md#agreements-and-licenses).
- [x] This pull request follows the contributing guidelines, recommended Git workflow, and templates.
- [x] I have tested my changes.
- [x] I have updated the relevant documentation.
---

**Description**

Changes proposed in this pull request:

- Simplify content of readme files to guide users properly to run Luigi

**Issue link**

### CODE REVIEW

File: core/examples/luigi-sample-angular/README.md [hunk 1]
```diff
@@ -1,25 +1,51 @@
...
+
+6. Reflect changes on luigi code on the example app.
+
+    a. For Luigi Core you have to let it bundle again on every change you apply to it. The easiest approach is to open Luigi´s root folder in another tab in your terminal window and have the following running: 
+    ```bash    
+    cd core && npm run bundle -- --watch
+    ```
+    b. Luigi Client is not bundled, so you are fine just updating it without bundling. 
+    
+    c. *The autoreload of your application only updates the application*. Type CMD + R to reflect the changes of the linked modules (Luigi Core and Luigi Client) on the website.
```
Comment (klaudiagrz) [inline core/examples/luigi-sample-angular/README.md]: We do not use italics in Kyma documentation.

``The auto-reload of your application only updates the application. Type `CMD + R` to reflect the changes of the linked modules (Luigi Core and Luigi Client) in the website.``

File: README.md [hunk 2]
```diff
@@ -15,28 +15,7 @@ Read them to learn more about each project in this repository.
...
-lerna bootstrap
-```
-
-3. Bundle cross-dependencies.
-```bash
-# Lerna runs the bundle script in every package where the script exists.
-
-lerna run bundle
-```
+The best way to see Luigi up and running and develop it is by using the [Angular example app](/core/examples/luigi-sample-angular/README.md).
```
[the remark this answers is not in the document]
Comment (tomekpapiernik) [reply README.md]: How about 
"To run Luigi and to develop it (...)"
?

[the remark this answers is not in the document]
Comment (maxmarkus) [reply README.md]: done

File: client/README.md [hunk 3]
```diff
@@ -7,14 +7,19 @@ This project contains a library that allows your application to use all features
...
 // es5
 var LuigiClient = require('@kyma-project/luigi-client').LuigiClient;
 
 // es6 and beyond
 import { LuigiClient } from '@kyma-project/luigi-client';`
-```
\ No newline at end of file
+```
+
+## See it in action
```
[the remark this answers is not in the document]
Comment (maxmarkus) [reply client/README.md]: done

File: client/README.md [hunk 4]
```diff
@@ -7,14 +7,19 @@ This project contains a library that allows your application to use all features
...
+
+You can see Luigi Client in action by running the [Angular example app](/core/examples/luigi-sample-angular/README.md).
```
[the remark this answers is not in the document]
Comment (maxmarkus) [reply client/README.md]: done

File: core/README.md [hunk 5]
```diff
@@ -8,51 +8,45 @@ Luigi core is a micro front-end framework written in Svelte. To learn how it wor
...
-**Build and deploy in development mode**
+1. Install Lerna globally.
+    ```bash
+    npm install -g lerna
+    ```
 
-For non-minified sources, use this approach.
-Go to the `core` folder and run the following command:
+2. Install dependencies. 
+The `lerna bootstrap` command executes npm install and links cross-dependencies.
```
[the remark this answers is not in the document]
Comment (maxmarkus) [reply core/README.md]: done

File: core/README.md [hunk 6]
```diff
@@ -8,51 +8,45 @@ Luigi core is a micro front-end framework written in Svelte. To learn how it wor
...
+    ```bash
+    lerna bootstrap
+    ```
 
-`npm run-script bundle-develop && cp -r ./public/ examples/luigi-sample-angular/node_modules/@kyma-project/luigi-core/`
+3. Bundle luigi core.
```
[the remark this answers is not in the document]
Comment (maxmarkus) [reply core/README.md]: done

File: core/README.md [hunk 7]
```diff
@@ -8,51 +8,45 @@ Luigi core is a micro front-end framework written in Svelte. To learn how it wor
...
+    lerna run bundle
 
-**Link and build with Lerna**
+    // or non-minified and watcher:
+    lerna run bundle-develop
+    ```
 
-Linking and building with Lerna is the preferred development method.
-This mono repository uses [Lerna](https://lernajs.io/) for project management. Check that Lerna is installed. If it is not, run `npm install --global lerna`
+### See it in action
```
[the remark this answers is not in the document]
Comment (maxmarkus) [reply core/README.md]: done

File: core/README.md [hunk 8]
```diff
@@ -8,51 +8,45 @@ Luigi core is a micro front-end framework written in Svelte. To learn how it wor
...
 
-Follow these steps:
+You can see Luigi Core in action by running the [Angular example app](/core/examples/luigi-sample-angular/README.md).
```
[the remark this answers is not in the document]
Comment (maxmarkus) [reply core/README.md]: done

File: core/examples/README.md [hunk 9]
```diff
@@ -3,7 +3,9 @@
 ## Overview
 
 This repository contains sample applications showing Luigi framework capabilities.
+Currently we provide an Angular example application. Contributions for more examples are always very welcome!
```
[the remark this answers is not in the document]
Comment (maxmarkus) [reply core/examples/README.md]: removed.

File: core/examples/README.md [hunk 10]
```diff
@@ -3,7 +3,9 @@
...
 
 ## Installation
 
-See particular subfolders for installation instructions.
+See particular subfolders for installation instructions:
```
[the remark this answers is not in the document]
Comment (maxmarkus) [reply core/examples/README.md]: done

File: core/examples/README.md [hunk 11]
```diff
@@ -3,7 +3,9 @@
...
+* [Angular example app](/core/examples/luigi-sample-angular/README.md)
```
[the remark this answers is not in the document]
Comment (maxmarkus) [reply core/examples/README.md]: done

File: core/examples/luigi-sample-angular/README.md [hunk 12]
```diff
@@ -1,25 +1,51 @@
-# Sample luigi sample application written in Angular
+# Sample luigi application written in Angular
```
[the remark this answers is not in the document]
Comment (maxmarkus) [reply core/examples/luigi-sample-angular/README.md]: done

File: core/examples/luigi-sample-angular/README.md [hunk 13]
```diff
@@ -1,25 +1,51 @@
...
 
 ## Development
```
[the remark this answers is not in the document]
Comment (maxmarkus) [reply core/examples/luigi-sample-angular/README.md]: I added an Overview section, would you provide a description please? 
This is an Angular based sample application which runs inside Luigi Framework. Therefore the steps are similar, bot not equal.
Luigi Core consists only of static pages, while this example shows the features that are currently possible with Luigi.

File: core/examples/luigi-sample-angular/README.md [hunk 14]
```diff
@@ -1,25 +1,51 @@
...
+    ```
 
-2. Bundle cross-dependencies.  
-`lerna run` runs the given script in every package referenced in lerna.json
-```bash
-lerna run bundle
-```
+2. Install dependencies.
+    ```bash
+    # Lerna bootstrap makes npm install and links cross-dependencies.
```
[the remark this answers is not in the document]
Comment (maxmarkus) [reply core/examples/luigi-sample-angular/README.md]: done

File: core/examples/luigi-sample-angular/README.md [hunk 15]
```diff
@@ -1,25 +1,51 @@
...
 
-## Run server
+    lerna bootstrap
+    ```
+
+3. Bundle luigi core.
```
[the remark this answers is not in the document]
Comment (maxmarkus) [reply core/examples/luigi-sample-angular/README.md]: done

File: core/examples/luigi-sample-angular/README.md [hunk 16]
```diff
@@ -1,25 +1,51 @@
...
+    ```
+
+4. Start the example application.
+    ```bash
+    npm start
+    ```
+
+5. Open it in your browser.
+
+    In your favourite browser, go to http://localhost:4200/
```
[the remark this answers is not in the document]
Comment (maxmarkus) [reply core/examples/luigi-sample-angular/README.md]: done

File: core/examples/luigi-sample-angular/README.md [hunk 17]
```diff
@@ -1,25 +1,51 @@
...
+
+6. Reflect changes on luigi code on the example app.
```
[the remark this answers is not in the document]
Comment (maxmarkus) [reply core/examples/luigi-sample-angular/README.md]: done

File: core/examples/luigi-sample-angular/README.md [hunk 18]
```diff
@@ -1,25 +1,51 @@
...
+
+    a. For Luigi Core you have to let it bundle again on every change you apply to it. The easiest approach is to open Luigi´s root folder in another tab in your terminal window and have the following running:
```
[the remark this answers is not in the document]
Comment (maxmarkus) [reply core/examples/luigi-sample-angular/README.md]: done

File: core/examples/luigi-sample-angular/README.md [hunk 19]
```diff
@@ -1,25 +1,51 @@
...
+
+    In your favourite browser, go to http://localhost:4200/
+
+6. Reflect changes on luigi code on the example app.
+
+    a. For Luigi Core you have to let it bundle again on every change you apply to it. The easiest approach is to open Luigi´s root folder in another tab in your terminal window and have the following running: 
+    ```bash    
+    cd core && npm run bundle -- --watch
+    ```
+    b. Luigi Client is not bundled, so you are fine just updating it without bundling.
```
[the remark this answers is not in the document]
Comment (maxmarkus) [reply core/examples/luigi-sample-angular/README.md]: done

File: core/examples/luigi-sample-angular/README.md (hunk 18 above)
[the remark this answers is not in the document]
Comment (maxmarkus) [reply core/examples/luigi-sample-angular/README.md]: What do you mean with "Change letters to the bullet points"?

File: core/examples/luigi-sample-angular/README.md (hunk 1 above)
Comment (maxmarkus) [reply core/examples/luigi-sample-angular/README.md]: done

File: core/examples/luigi-sample-angular/README.md [hunk 20]
```diff
@@ -1,25 +1,55 @@
...
-```bash
-lerna bootstrap
-```
+    - You must let the Luigi core to bundle every change you apply to it. The easiest approach is to open the Luigi root folder in another tab of your terminal window and run the following command: 
+    ```bash    
+    cd core && npm run bundle -- --watch
+    ```
+    - The Luigi Client is not bundled, so you are able to update it without bundling.
+    
+    - The auto-reload of your application only updates the application. Type `CMD + R` to reflect the changes of the linked modules (Luigi Core and Luigi Client) in the website.
```
Comment (klaudiagrz) [inline core/examples/luigi-sample-angular/README.md]: `` Type `CMD + R` to reflect the changes of the linked Luigi Core and Luigi Client modules in the website.``

File: core/examples/luigi-sample-angular/README.md [hunk 21]
```diff
@@ -1,25 +1,55 @@
...
+    ```
+
+4. Start the example application.
+    ```bash
+    npm start
+    ```
+
+5. Open it in your browser.
+
+    Open it in your browser by going to [http://localhost:4200/](http://localhost:4200/)`.
```
[the remark this answers is not in the document]
Comment (maxmarkus) [reply core/examples/luigi-sample-angular/README.md]: Done

**Pull request merged**